pkg/consts/align/align.go

```diff
@@ -1,4 +1,4 @@
-// Package contains all align types.
+// Package align contains all align types.
 package align
 
 // Type is arRepresentation of a column align.
```

Comment:
Could you fix this typo? // This is a representation

Reply:
> Could you fix this typo? // This is a representation

Rename the package name to `represents` ？